test/api_users_test.go

```diff
@@ -56,15 +56,15 @@ func createRole(name string, description string, roleType string) *client.Create
 		newRole = &wrapper
 	}
 	if roleType == "custom" {
-		env := "*"
+		env := "development"
```

Comment:
This is now optional on the Unleash side and if it's * then it raises an error. So I've set it to an environment that Unleash likes